docs/mode-reference.rst

```diff
@@ -23,29 +23,81 @@ Types of attributes values in this reference:
 +------------+-------------------------------------------------------------------------------------+
 
 
-Attributes
-----------
+Language Only Attributes
+------------------------
+
+These attributes are only valid at the language level (ie, they many only exist on the top-most language object and have no meaning if specified in children modes).
+
+
+name
+^^^^
+
+- **type**: string
+
+The canonical name of this language, ie "JavaScript", etc.
+
 
 case_insensitive
 ^^^^^^^^^^^^^^^^
 
-**type**: boolean
+- **type**: boolean
 
 Case insensitivity of language keywords and regexps. Used only on the top-level mode.
 
 
 aliases
 ^^^^^^^
 
-**type**: array
+- **type**: array
 
 A list of additional names (besides the canonical one given by the filename) that can be used to identify a language in HTML classes and in a call to :ref:`getLanguage <getLanguage>`.
 
 
+classNameAliases
```

Comment:
@allejo Any thoughts on this naming?  It seems clear to me... Other ideas were nesting, but that seems more complex:

```ts
themes: { aliases: {}}
```

Reply:
I'd be fine with `classNameAliases`. Another suggestion would be `styleClassAliases` (or even `styleAliases`) which makes it a bit clearer what we're talking about. You should decide this having the newbie user in mind.